index.html

```diff
@@ -1,85 +1,78 @@
+      href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
+    />
 
-<head>
-  <meta charset="UTF-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-  <title>Book Listed</title>
-  <!-- Compiled and minified CSS -->
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css" />
+    <!-- Compiled and minified JavaScript -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
 
-  <!-- Compiled and minified JavaScript -->
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+    <!--Import Google Icon Font-->
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
 
-  <!--Import Google Icon Font-->
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!--Import materialize.css-->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
+    />
+  </head>
 
-  <!--Import materialize.css-->
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
-</head>
-
-<body>
-  <nav>
-    <div class="nav-wrapper top">
-      <div class="row">
-        <a href="" class="brand-logo center name">BookListed</a>
-      </div>
+  <body>
+    <nav>
+      <div class="nav-wrapper top">
+        <div class="row">
+          <a href="" class="brand-logo center name">BookListed</a>
+        </div>
 
-      <div class="row center summary ">
-        Search NYTimes best sellers lists
+        <div class="row center summary ">
+          Search NYTimes best sellers lists
+        </div>
       </div>
-    </div>
-  </nav>
-  <div class="container">
+    </nav>
+    <div class="container">
       <div class="row">
-          <div class="col s3">
-              <div class="col s6 mainlogo">
-              </div>
-          </div>
+        <div class="col s3">
+          <div class="col s6 mainlogo"></div>
+        </div>
       </div>
       <div class="row">
-          <div class="col s3">
-          </div>
-          <div class="col s6">
-              <div class="row">
-                  <div class="input-field col s12">
-                      <i class="material-icons prefix">library_books</i>
-                      <input type="text" id="autocomplete-input" class="autocomplete">
-                      <label for="autocomplete-input">Pick a Category</label>
-                      <input type="text" class="datepicker">
-                  </div>
-              </div>
-          </div>
-      </div>
-  </div>
-  <div>
-    <div class="container">
-      <div class="col s12 m7">
-        <div class="card horizontal">
-          <div class="card-image">
```

Comment:
I have used this card as a multiline string in app.js

Reply:
This is awesome! Good thinking.